guidelines/decision-process.md

```diff
@@ -1 +1,23 @@
+# Decision making process
 
+## Preamble
+The Chair Group discussion led to a consensus to start with a light weight consensus-based process that would eventually use voting as a last resort method in case of impasse.. The group could eventually be in need to defend its decision making process against trolls, yet at the moment all group members are considered in good faith, so the working group reserves the option to specify disciplinary actions against trolls as part of its guidelines at a later stage..
+
+## Definitions
+1. *Consensus* is defined as lack of sustained opposition.
+2. *Good standing* is a characteristic of group members who fullfill their implicit and explicit obligations and hence are allowed to excercise all of their rights within the group without restriction.
+3. *Blacklist* (*blacklisted*, *blacklisting*) is a taxative enumeration of group mebers who are temporarily excluded from excercising their rights within the group.
+4. *Normative keywords* as defined in [BCP 14](https://tools.ietf.org/html/bcp14)
+
+## Rules
+1. Any current group member is deemed to be in good standing unless blacklisted.
```

Comment:
Instead of Rules probably i would call it "Process" what do you think ?

Reply:
I think that it's indeed rules rather than process steps..